linux_os/guide/system/permissions/partitions/mount_option_home_nosuid/tests/fstab.fail.sh

```diff
@@ -1,4 +1,9 @@
 #!/bin/bash
+{{% if "ol" in product %}}
+# platform = multi_platform_example
```

Comment:
Hum, maybe it was done this way because these tests scenarios are failing. They are failing because the remediation is aborted when the home directory does not exist, which doesn't seem ideal. It would be better to make the remediation more robust instead of skipping test scenario scripts.

Reply:
Well I made an equivalent test for the template, and I just thought it would be better to use this new generalized test

Reply:
The case is that the templated script and its supposedly equivalent in the rule level behave differently. So, they are not equivalent. The existing tests should not fail. The existing test scenarios are failing because the remediation is not capable to fix the partition settings if there isn't a home dir created.

Reply:
Well is equivalent in the sense that the conf in fstab is what makes OVAL fail. This test is not correctly set for the template, precisely because it has a different behavior, if there are no users with home directory under `/home` the test won't result in fail, and that is expected.

Even if we want to fix that in remediation, I think we should set a specific test to validate more explicitly that scenario